pkg/model/openstackmodel/firewall.go

```diff
@@ -21,6 +21,7 @@ import (
 	"k8s.io/kops/pkg/apis/kops"
 	"k8s.io/kops/upup/pkg/fi"
 	"k8s.io/kops/upup/pkg/fi/cloudup/openstacktasks"
+
```

Comment:
We probably shouldn't do this in this PR (because it's not thematically related).  (but not a big deal)

Reply:
Done, my installed goimports was messing this up for some reason, I removed it from the patch. (same as above)